action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: 'Pipeline that will be run through VIB. This path is relative to the config folder.'
     required: false
     default: 'vib-pipeline.json'
+  artifacts:
```

Comment:
The naming is too ambiguous here. I think we should change it to something like `upload-artifacts` instead.